### PR TITLE
Updates balloon alerts in the aiming component

### PR DESCRIPTION
**Pull request by benbot16**
## About The Pull Request

This does two things:
1. It shortens the balloon alert displayed when aiming a gun at someone.
2. It makes commands given while aiming actually use says instead of balloon alerts.

Each possible command has a few different options to pick from, which hopefully shouldn't make it feel too samey. I can change the options or add more options if anyone has suggestions, though.

I didn't touch some of the other alerts, but I can change them if that's needed.

## Why It's Good For The Game

See #7164, but basically when I originally changed the text in aiming from says to balloon alerts, I made them way too long since I modeled them after other balloon alerts that were also way too long.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
Giving a command while aiming

</details>

## Changelog
:cl:
tweak: Changes/replaces some of the balloon alerts displayed when aiming a gun at someone.
/:cl:

### CODE REVIEW

File: code/datums/components/aiming.dm [hunk 1]
```diff
@@ -38,8 +38,8 @@
 	user.visible_message("<span class='warning'>[user] points [parent] at [target]!</span>")
 	to_chat(target, "<span class='userdanger'>[user] is pointing [parent] at you! If you equip or drop anything they will be notified! \n<b>You can use *surrender to give yourself up</b>.</span>")
 	to_chat(user, "<span class='notice'>You're now aiming at [target]. If they attempt to equip anything you'll be notified by a loud sound.</span>")
-	user.balloon_alert_to_viewers("[user] points [parent] at [target]!", ignored_mobs = list(user, target))
-	user.balloon_alert(target, "[user] points [parent] at you!")
+	user.balloon_alert_to_viewers("[user] points a gun at [target]!", ignored_mobs = list(user, target))
+	user.balloon_alert(target, "[user] points a gun at you!")
```
Comment (PowerfulBacon) [inline code/datums/components/aiming.dm]: I think this is better as [parent] in cases where the gun is something weird, like a magic wand

File: code/datums/components/aiming.dm [hunk 2]
```diff
@@ -192,16 +190,11 @@ AIMING_DROP_WEAPON means they selected the "drop your weapon" command
 			fire()
 			return
 		if(RAISE_HANDS)
-			alert_message = "raise your hands!"
-			alert_message_3p = "raise their hands!"
+			user.say(pick(list("Put your hands above your head!", "Hands! Now!", "Hands up!")), forced = "Weapon aiming")
```
Comment (PowerfulBacon) [inline code/datums/components/aiming.dm]: Byond supports pick from parameters, a list input isn't required
image

File: code/datums/components/aiming.dm (hunk 1 above)
Comment (benbot16) [reply code/datums/components/aiming.dm]: Alright, reverted it, although I do think it feels a bit clunky in cases where there's wielded weapons with long names.

File: code/datums/components/aiming.dm (hunk 2 above)
Comment (benbot16) [reply code/datums/components/aiming.dm]: Removed the lists.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**